APMrover2/defines.h

```diff
@@ -30,6 +30,7 @@
 #define LOG_STEERING_MSG        0x0D
 #define LOG_GUIDEDTARGET_MSG    0x0E
 #define LOG_WHEELENCODER_MSG    0x0F
+#define LOG_CRUISELEARN_MSG     0x10
```

Comment:
Suggest unconditionally logging this.  cruise learn is something the user triggers, and we can't really afford to lose logging bits...